not-paid.js

```diff
@@ -18,7 +19,9 @@
 		if(opacity >= 0 && opacity <= 1) {
 			document.getElementsByTagName("BODY")[0].style.opacity = opacity;
 		}
-		
+		if(opacity == 0 && deadline_message != "") {
```

Comment:
Empty strings evaluate to boolean false anyway (and opacity `0`). This could be rewritten as:

```suggestion
		if(!opacity && deadline_message) {
```

https://stackoverflow.com/questions/8692982/in-javascript-is-an-empty-string-always-false-as-a-boolean

Reply:
Ah, thanks!  I knew there was probably an easier way of doing this, I just threw something together, and I haven’t done much JavaScript lately.